README.md

```diff
@@ -1,2 +1,77 @@
 # krux-marathon-api
-krux marathon api in python
+
+This is a command line tool to help create, delete, and keep Marathon apps in a specified state.
+It comes with a sample json config file for specifying state of an app.
+
+The Application defaults to localhost and port 8080 use the host and port flags to change the location.
+
+```./cli.py --host marathon_server.com --port 8081 --list_apps```
+
+## Create / Update Marathon Application
+Requires a valid json file (see sample).
+
+```./cli.py --config_file ../bash_nc_listen_sample.json```
```

Comment:
Also, doesn't it make more sense to pass `--create` or take `action` positional argument? You can refer to [`krux-dns`](https://github.com/krux/python-krux-dns).